docs/examples/mht_example.py

```diff
@@ -0,0 +1,248 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+"""
+========================================================
+General Multi Hypotheses tracking implementation example
+========================================================
+"""
+
+# %%
+# Multi hypotheses tracking (MHT) algorithm is considered one of the best algorithm
+# for visual tracking, which consists in creating a tree of potential tracks for
+# each target candidate (in a multi-target scenario) and prune such hypotheses
+# in the data association phase. It is particularly efficient in maintain trajectories of
+# multiple objects and handling uncertainties and ambiguities of tracks (e.g. presence of
+# clutter).
+# MHT, by definition, has several algorithms that fall under this definition, which
+# we can list as Global Nearest Neighbour (GNN), Joint Probabilistic Data association
```

Comment:
```suggestion
# include Global Nearest Neighbour (GNN), Joint Probabilistic Data association
```
There are a fair few others, no? It might be worth mentioning that these are implemented in Stone Soup.